homeassistant/components/blebox/cover.py

```diff
@@ -14,7 +15,13 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
-from .const import BLEBOX_TO_HASS_COVER_STATES, BLEBOX_TO_HASS_DEVICE_CLASSES
+from .const import BLEBOX_TO_HASS_COVER_STATES
```

Comment:
I think `BLEBOX_TO_HASS_COVER_STATES` should also be moved to `cover.py`
It is only relevant here.